playkit/src/main/java/com/kaltura/playkit/ads/PKAdEventListener.java

```diff
@@ -0,0 +1,10 @@
+package com.kaltura.playkit.ads;
+
+import com.kaltura.playkit.PKEvent;
+
+/**
+ * Created by gilad.nadav on 25/01/2017.
+ */
+public interface PKAdEventListener {
```

Comment:
I think this should be more specific.
```java
interface AdProviderListener {
  void onAdStarted(AdInfo);
  void onAdCuePointsChanged(AdCuePoints);
  void onAdRequestFailed(..);
}
```

The way it's now is confusing: PKEvent is normally used with the MessageBus.